cpp/test/prims/batched/batched_csr.cu

```diff
@@ -0,0 +1,208 @@
+/*
```

Comment:
nitpick: since this file is already under `batched` folder, maybe just call this one as `csr.cu`?